apps/masterbots.ai/components/routes/chat/chat-message.tsx

```
@@ -1,35 +1,59 @@
 import remarkGfm from 'remark-gfm'
 import remarkMath from 'remark-math'
-
-//* Define the props interface for the ChatMessage component
-export interface ChatMessageProps {
-  message: Message // The chat message to display
-  sendMessageFromResponse?: (message: string) => void // Callback to send a new message
-  chatbot?: Chatbot // Chatbot configuration
-  actionRequired?: boolean // Whether to show message actions
-}
+import { useState } from 'react'
+import { ChatMessageProps, WebSearchResult } from '@/types/types'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)